client/me/purchases/concierge-banner/index.jsx

```diff
@@ -88,13 +88,10 @@ class ConciergeBanner extends Component {
 
 			case CONCIERGE_HAS_AVAILABLE_PURCHASED_SESSION:
 				headerText = translate( 'Our experts are waiting to help you' );
-				mainText = translate(
-					'Schedule your 45-minute 1-1 Quick Start Session with a Happiness Engineer!',
-					{
-						comment:
-							'Quick Start Session is a one-on-one video session between the user and our support staff.',
-					}
-				);
+				mainText = translate( 'Schedule your 1-1 Quick Start Session with a Happiness Engineer!', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Schedule your 45-minute 1-1 Quick Start Session with a Happiness Engineer!' )` *ES Score: 7*